libraries/AP_NavEKF3/AP_NavEKF3_Outputs.cpp

```diff
@@ -308,6 +308,7 @@ bool NavEKF3_core::getLLH(struct Location &loc) const
                 loc.lng = EKF_origin.lng;
                 loc.offset(lastKnownPositionNE.x + posOffsetNED.x,
                            lastKnownPositionNE.y + posOffsetNED.y);
+                loc.alt = EKF_origin.alt + lastKnownPositionD;
```

Comment:
probably - lastKnownPositionD  as  alt  is UP  oriented  and lastKnownPositionD   DOWN oriented

Reply:
Fixed, thanks.